junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/builder/JUnitReportBuilder.java

```diff
@@ -44,25 +44,37 @@ public JUnitReportBuilder(List<String> xmlReports, String targetBuildPath) throw
         return processedTestSuites;
     }
 
-    private List<TestSuiteModel> processXmlReports(List<String> xmlReports) throws JAXBException {
-        JAXBContext cntx = JAXBContext.newInstance(TestSuiteModel.class);
-
-        Unmarshaller unm = cntx.createUnmarshaller();
+    private void processSuite(TestSuiteModel ts) {
+        for (TestCaseModel tc : ts.getTestcase()) {
+            if (tc.getError() != null) {
+                System.out.println();
```

Comment:
I've just moved some code from one line to another; it was already here. But it looks unneeded in any case indeed.